src/utils/microfmt.cpp

```diff
@@ -0,0 +1,14 @@
+#include "utils/microfmt.hpp"
+
+#include <iostream>
+
+namespace cpptrace {
+namespace detail {
+
+    std::ostream& get_cout()
+    {
```

Comment:
Minor formatting thing
```suggestion
    std::ostream& get_cout() {
```

Reply:
Will fix